Cargo.toml

```diff
@@ -55,3 +55,15 @@ default-members = [
 ring = { git = "https://github.com/crypto-com/ring.git", rev = "bdbcc7041095f028d49d9fecd7edcf26d6083274" }
 # FIXME: use upstream when merged
 sha2 = { git = "https://github.com/crypto-com/hashes.git", rev = "289d5b76f2163a3808010341ed1df3cb156d97e1" }
+
+[patch.crates-io.tendermint]
+git = "https://github.com/informalsystems/tendermint-rs.git"
+branch = "master"
+
+[patch.crates-io.tendermint-rpc]
+git = "https://github.com/informalsystems/tendermint-rs.git"
+branch = "master"
+
+[patch.crates-io.tendermint-light-client]
+git = "https://github.com/informalsystems/tendermint-rs.git"
+branch = "master"
```

Comment:
it's now published on crates.io https://crates.io/crates/tendermint-light-node ?

Reply:
https://crates.io/crates/tendermint-light-client
https://crates.io/crates/tendermint
https://crates.io/crates/tendermint-rpc

Reply:
done, removed patch.